test/screenshot/infra/lib/selenium-api.js

```diff
@@ -972,6 +977,25 @@ class SeleniumApi {
     return `${this.cli_.colorizeUrl(url)} > ${userAgentColor(userAgent.alias)}`;
   }
 
+  /**
+   * @return {boolean}
+   * @private
+   */
+  isBusinessHours_() {
+    const MORNING_UTC_HOURS = 13; // 1pm UTC === 9am EST
+    const EVENING_UTC_HOURS = 1; // 1am UTC === 6pm PST
+    const SATURDAY = 7;
```

Comment:
Saturday is 6, not 7. (There are not 8 days in a week, as much as we might wish there were.)

Reply:
Good catch. I'll create a follow up PR. Thanks!